usaspending_api/references/management/commands/load_city_county_state_code.py

```diff
@@ -0,0 +1,209 @@
+import logging
+import zipfile
+
+from contextlib import contextmanager
+from django.core.management.base import BaseCommand
+from django.db import transaction, connections, DEFAULT_DB_ALIAS
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from usaspending_api.common.etl import ETLTable, ETLTemporaryTable, operations
+from usaspending_api.common.helpers.sql_helpers import execute_dml_sql
+from usaspending_api.common.helpers.timing_helpers import Timer
+from usaspending_api.common.retrieve_file_from_uri import RetrieveFileFromUri
+from usaspending_api.common.zip import extract_single_file_zip
+from usaspending_api.etl.operations.subaward.update_city_county import update_subaward_city_county
+from usaspending_api.references.models import CityCountyStateCode
+
+
+logger = logging.getLogger("console")
```

Comment:
```suggestion
logger = logging.getLogger("script")
```

Better log format for management commands